src/edu/csus/ecs/pc2/core/scoring/DefaultScoringAlgorithm.java

```diff
@@ -626,6 +647,20 @@ private void createStandingXML (TreeMap<StandingsRecord, StandingsRecord> treeMa
             groupRank[i] = 0;
             groupIndexRank[i] = 0;
         }
+        long[] divisionNumSolved = new long[divisionCount];
+        for (int i = 0; i < divisionCount; i++) {
```

Comment:
The use of "long" here seems to imply that "int" wouldn't be enough -- but that would imply that there could be more than 2^32 solutions in a Division.  Seems like overkill...